src/main/kotlin/racingcar_winner/model/Cars.kt

```diff
@@ -1,47 +1,28 @@
 package racingcar_winner.model
 
-import racingcar_winner.util.progressNumberToBar
-
-class Cars(
-    cars: List<Car>
+public class Cars private constructor(
+    private val cars: List<Car>
 ) {
-    private val cars: List<Car> = cars.toList()
-
-    fun getNumberOfCars(): Int = cars.size
-
     fun moveAll(randomNumberFactory: RandomNumberManager) {
         for (car in cars) {
             val randomNumber = randomNumberFactory.generateRandomNumber()
             car.tryToMoveForward(randomNumber)
         }
     }
 
-    fun getProgressOfCars(): List<String> {
-        val cars = cars.toList()
-        return cars.map { car ->
-            "${car.name} : ${car.progress.progressNumberToBar()}"
-        }
+    fun toList(): List<Car> {
+        return cars.toList()
     }
 
-    fun getWinners(): List<String> {
-        val cars = cars.toList()
-        val highScore = getHighScore()
-
-        return cars.filter { car ->
-            car.progress == highScore
-        }.map {
-            it.name
+    companion object {
+        fun makeCars(cars: List<Car>): Cars {
+            return Cars(cars)
         }
```

Comment:
외부에 내부 컬렉션을 공개해줄 때만 아니라 외부에서 받을 때에도 방어적 복사를 해주는 것이 안전합니다.
```suggestion
        fun makeCars(cars: List<Car>): Cars {
            return Cars(cars.toList())
        }
```